### PR TITLE
Added integrating with CI topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/integrate-with-other-tools/integrate-with-ci-systems/

Preview at https://openshift-docs-git-integrating-ci-gnelson.vercel.app/openshift-acs/master/integration/integrate-with-ci-systems.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer Please review after https://github.com/openshift/openshift-docs/pull/33863 is merged.

**Comment by bergerhoffer** [comment]
This one has a conflict if you can resolve

File: modules/integrate-circle-ci.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// * integration/integrate-with-ci-systems.adoc
+:_module-type: PROCEDURE
+[id="integrate-circle-ci_{context}"]
+= Using CircleCI
+
+You can integrate {product-title} with CircleCI.
+
+.Prerequisetes
+* You must have a token with `read` and `write` permissions for the `Image` resource.
+* Useranme and password of your DockerHub account.
```
Comment (bobfuru) [inline modules/integrate-circle-ci.adoc:12]: For consistency, I'd start both with "You have a..."

Comment (bobfuru) [inline modules/integrate-circle-ci.adoc:12]:
```suggestion
* You have a token with `read` and `write` permissions for the `Image` resource.
* You have a username and password for your Docker Hub account.
```

File: modules/integrate-circle-ci.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.Procedure
+. Log in to CircleCI and open an existing project or create a new project.
+. Click *Project Settings*.
+. Click *Environment variables*.
+. Click *Add variable* and create the following three environment variables:
+** *Name*: *STACKROX_CENTRAL_HOST* - The DNS name or IP address of Central.
+** *Name*: *ROX_API_TOKEN* - The API token to access {product-title}.
+** *Name*: *DOCKERHUB_PASSWORD* - The password for your DockerHub account.
+** *Name*: *DOCKERHUB_USER* - The username for your DockerHub account.
```
Comment (bobfuru) [inline modules/integrate-circle-ci.adoc:22]: I believe Docker Hub is two words.

File: modules/integrate-circle-ci.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+. Create a directory called `.circleci` in the root directory of your local code repository for your selected project, if you do not already have a CircleCI configuration file.
+. Create a `config.yml` configuration file with the following lines in the `.circleci` directory.
```
Comment (bobfuru) [inline modules/integrate-circle-ci.adoc:24]: Can you please change the period to a colon after directory? That's generally how we introduce an example code block.

**Review by bobfuru — COMMENTED** [review]
Overall, some nitpicky items based on IBM and OpenShift Docs style guides, along with some modularization conventions/questions about breaking some individual procedures into separe module files. I'm guessing a few of my comments have already been raised/addressed by other peer reviews, so please disregard if a decision has already been reached in those cases.

Overall, I hope you'll find my comments useful and helpful but feel free to reach out with any questions or need for clarification. Thanks for letting me bit a part of this process!

File: modules/integrate-circle-ci.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/integrate-circle-ci.adoc:12]: Also Useranme -> Username

**Comment by bobfuru** [comment]
@gaurav-nelson One more thing I forgot to mention is that where I've used the "Suggested change" feature, please be sure NOT to click "Commit suggestion" as that will start creating too many commits to keep track of. Some of us use the feature as a nice visual representation of a suggestion and it's pretty slick...but you will want to manually apply the changes. Thanks!

File: integration/integrate-with-ci-systems.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+== Integrating with your CI pipeline
+
+Once you have finished these procedures, the next step is to integrate with your CI pipeline.
+
+Each CI system may require a slightly different configuration.
+
+[id="using-jenkins"]
+=== Using Jenkins
+
+Use the link:https://plugins.jenkins.io/stackrox-container-image-scanner[StackRox Container Image Scanner] Jenkins plugin for integrating with Jenkins.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply integration/integrate-with-ci-systems.adoc:88]: Yeah, they still have to reanme that. Its correct for now.

File: modules/create-new-system-policy.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+. Select a notifier from the *Notifications* drop-down menu that receives alert notifications when a violation occurs for this policy.
++
+[NOTE]
+====
+You must integrate {product-title} with your notification providers, for example, webhooks, Jira, or PagerDuty, to receive alert notifications. Notifiers only show up if you have integrated any notification providers with {product-title}.
+====
+. Use *Restrict to Scope* to enable this policy only for a specific cluster, namespace, or label.
+You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+. Use *Exclude by Scope* to exclude deployments, clusters, namespaces, and labels.
+It means that the policy will not apply to the entities that you specify.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/create-new-system-policy.adoc:37]: No they are unrelated. Notifier is to specify which channel (Slack, email, etc. ) to use for notification. Whereas exclude and restrict applies to namespaces or cluster to apply or exclude the policy.

File: modules/create-new-system-policy.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+However, you cannot use regular expressions for selecting deployments.
+. For *Excluded Images (Build Lifecycle only)*, select all the images from the list for which you do not want to trigger a violation for the policy.
++
+[NOTE]
+====
+The *Excluded Images (Build Lifecycle only)* setting only applies when you check images in a continuous integration system (the Build lifecycle stage).
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/create-new-system-policy.adoc:44]: Yeah, UI text.

File: modules/create-new-system-policy.adoc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+It does not have any effect if you use this policy to check running deployments (the Deploy lifecycle stage) or runtime activities (the Runtime lifecycle stage).
+====
+. In the *Policy Criteria* section, configure the attributes on which you want to trigger the policy.
+//TODO: Add link to policy crietria
+. Select *Next* on the panel header.
+. The new policy panel shows a preview of the violations that get triggered if you enable the policy.
+. Select *Next* on the panel header.
+. Choose the enforcement behavior for the policy.
+It is only available for the stages you select when configuring *Lifecycle Stages*.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/create-new-system-policy.adoc:53]: I've changed it to:
```
Enforcement settings are only available for the stages that you selected for the *Lifecycle Stages* option. 
```

File: modules/create-new-system-policy.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (bobfuru) [reply modules/create-new-system-policy.adoc]: Cool, thanks. Your update makes sense.

File: modules/integrate-circle-ci.adoc (hunk 2 above)
Comment (bobfuru) [reply modules/integrate-circle-ci.adoc]: Looks like the first one (password) could be split into two words as well.

**Review by bobfuru — COMMENTED** [review]
One more small nit and otherwise this LGTM and should be about ready for merge. 🎉

**Comment by gaurav-nelson** [comment]
@bobfuru Thank you for looking at this. Fixed that last nit. Ready to merge.

**Pull request merged**